MAUI/Popup/popup-positioning.md

```diff
@@ -38,6 +38,12 @@ Following are the list of options available to show SfPopup at various positions
 <tr>
 <td>{{'[Show(bool)](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_Show_System_Boolean_)'| markdownify }}</td>
 <td>Similar as <code>SfPopup.IsFullScreen</code> property.</td></tr>
+<tr>
+<td><code>SfPopup.Show(string title, string message)</code></td>
+<td>Static method to show popup with title and message.</td></tr>
+<tr>
+<td><code>SfPopup.Show(string title, string message, string acceptText)</code></td>
+<td>Static method to show popup with title, message and acceptText.</td></tr>

```

Comment:
<td>Static method to show a popup with the title, message, and acceptText.</td></tr>